### PR TITLE
runfix: Add Logarithmic Round to events data

### PR DESCRIPTION
**Pull request by thisisamir98**

### CODE REVIEW

File: src/script/util/NumberUtil.ts [hunk 1]
```diff
@@ -38,3 +38,14 @@ export const rootMeanSquare = (floatArray: number[] | Float32Array): number => {
   const sum = (floatArray as number[]).reduce((power, number) => power + number ** 2, 0);
   return Math.sqrt(sum) / floatArray.length;
 };
+
+/**
+ * The rounding is logarithmic, meaning that small numbers will on be slightly
```
Comment (ffflorian) [inline src/script/util/NumberUtil.ts]:
```suggestion
 * The rounding is logarithmic, meaning that small numbers will be slightly
```

File: src/script/util/NumberUtil.ts [hunk 2]
```diff
@@ -38,3 +38,14 @@ export const rootMeanSquare = (floatArray: number[] | Float32Array): number => {
...
+ * rounded (if at all) whereas larger numbers will be rounded more.
+ * This kind of rounding is roughly equivalent to having buckets with increasing size.
+ * @param exactValue number to be rounded
+ * @param factor determines by how much the value is rounded
+ */
+export const roundLogarithmic = (exactValue: number, factor: number) => {
```
Comment (ffflorian) [inline src/script/util/NumberUtil.ts]:
```suggestion
export const roundLogarithmic = (exactValue: number, factor: number): number => {
```

File: src/script/tracking/UserData.ts [hunk 3]
```diff
@@ -18,8 +18,8 @@
...
 
 export const UserData = {
-  CONTACTS: 'user.contacts',
-  IS_TEAM: 'team.is_team',
-  TEAM_SIZE: 'team.team_size',
-  USER_TYPE: 'team.user_type',
+  CONTACTS: 'user_contacts',
+  IS_TEAM: 'team_is_team',
+  TEAM_SIZE: 'team_team_size',
+  USER_TYPE: 'team_user_type',
```
Comment (ffflorian) [inline src/script/tracking/UserData.ts]: Are these changes part of logarithmic rounding?

**Review by ffflorian — CHANGES_REQUESTED** [review]

File: src/script/tracking/UserData.ts (hunk 3 above)
Comment (thisisamir98) [reply src/script/tracking/UserData.ts]: nope, these are updates by Eva

Comment (ffflorian) [reply src/script/tracking/UserData.ts]: Shouldn't we then apply them in a different PR?

Comment (thisisamir98) [reply src/script/tracking/UserData.ts]: done in https://github.com/wireapp/wire-webapp/pull/9581/commits/bec95acaa144272580fa0d1841b782a5e477f9e1

**Review by ffflorian — APPROVED** [review]

**Pull request merged**